src/main/java/org/sonarlint/intellij/fs/EditorFileChangeListener.kt

```diff
@@ -94,11 +94,6 @@ class EditorFileChangeListener(private val fileEventsNotifier: ModuleFileEventsN
             }
     }
 
-    // SLI-551 Only send events on .py files (avoid parse errors)
-    // For Rider, send all events for OmniSharp
-    private fun shouldNotify(file: VirtualFile) = file.isValid && (SonarLintUtils.isRider() || file.path.endsWith(".py"))
```

Comment:
I think this check disappeared. I agree filtering is not needed for the backend, but if we want to keep the same behaviorr, we should keep it to notify the engine only for the relevant files